server/kafka/kafkaproducer.go

```diff
@@ -0,0 +1,69 @@
+package kafka
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/cosminrentea/gobbler/server/configstring"
+	"github.com/cosminrentea/gobbler/server/service"
+	"time"
+)
+
+type Producer interface {
+	service.Startable
+	service.Stopable
+	Report(topic string, bytes []byte, key string)
```

Comment:
interesting, I will take a look and try to change the func signature in the next PR, which will contain also more tests